library/control/src/modules/ProductFeatures.rb

```diff
@@ -93,7 +93,8 @@ def main
           "addon_selections"                     => [],
           "inform_about_suboptimal_distribution" => false
         },
-        "network"      => { "force_static_ip" => false }
+        "network"                  => { "force_static_ip" => false },
+        "configuration_management" => {}
```

Comment:
Does this mean there are **no** defaults? I doubt that. We should add a comment pointing to the place where the defaults are defined.